### PR TITLE
[CUBRIDQA-860] add rqg test guide

### PR DESCRIPTION
**Pull request by slamdunkorchid**
refer to http://jira.cubrid.org/browse/CUBRIDQA-860

### CODE REVIEW

File: doc/rqg_guide.md [hunk 1]
```diff
@@ -0,0 +1,2823 @@
+# 1. Test Objective
+This guide is introduced to how to perform a RQG test. The RQG test is amed to test CUBRID stability especially in terms of server crash. It generates various of querys(update/delete/insert/select) automatically based on grammar files and executes these queries in parallel.
+# 2. Test Tools
+## 2.1 CTP 
+The RQG test is an extension of shell test. RQG test cases are executed via CTP.
+
+## 2.2 random_query_generator 
+It is an external tool provided by MySQL.
```
Comment (fanzaiqiang) [inline doc/rqg_guide.md]:
```suggestion
It is an external tool. Test case can find it via `RQG_HOME` environmental variable.
```

File: doc/rqg_guide.md [hunk 2]
```diff
@@ -0,0 +1,2823 @@
+# 1. Test Objective
+This guide is introduced to how to perform a RQG test. The RQG test is amed to test CUBRID stability especially in terms of server crash. It generates various of querys(update/delete/insert/select) automatically based on grammar files and executes these queries in parallel.
```
Comment (fanzaiqiang) [inline doc/rqg_guide.md]:
```suggestion
This guide is to introduce RQG test. The RQG test is amed to test CUBRID stability especially in terms of server crash. It randomly generates various of queries (UPDATE/DELETE/INSERT/SELECT and others) automatically based on grammar files and executes these queries with many threads.
```

File: doc/rqg_guide.md [hunk 3]
```diff
@@ -0,0 +1,2823 @@
+# 1. Test Objective
+This guide is introduced to how to perform a RQG test. The RQG test is amed to test CUBRID stability especially in terms of server crash. It generates various of querys(update/delete/insert/select) automatically based on grammar files and executes these queries in parallel.
+# 2. Test Tools
```
Comment (fanzaiqiang) [inline doc/rqg_guide.md]: Merge `#2` and `#3` chapters.
And adjust order as first installation, then usage.

**Comment by slamdunkorchid** [comment]
Please continue to review

**Pull request merged**